src/test/fuzz/descriptor_parse.cpp

```diff
@@ -108,6 +108,25 @@ class MockedDescriptorConverter {
 //! The converter of mocked descriptors, needs to be initialized when the target is.
 MockedDescriptorConverter MOCKED_DESC_CONVERTER;
 
+//! Maximum number of derivation indexes for an extended key's derivation path in a descriptor.
+static int MAX_DEPTH{2};
+
+/** Whether the buffer, if it represents a valid descriptor, contains a derivation path which depth
+ * is larger than the maximum authorized. Note this may also be hit for deriv paths in origins. */
+static bool HasDeepDerivPath(const FuzzBufferType& buff)
```

Comment:
Correct me if I'm wrong, but wouldn't `HasDeepDerivPath` avoid descriptors like `pkh([d34db33f/44'/0'/0']xpub6ERApfZwUNrhLCkDtcHTcxd75RbzS1ed54G1LkBUHQVHQKqhMkhgbmJbZRkrgZw4koxb5JaHWkY4ALHY2grBGRjaDMzQLcgJvLJuZZvRcEL/1/*)`? Is this intended?

Reply:
Yes:
> Note this may also be hit for deriv paths in origins.

Reply:
Thanks